setup.cfg

```diff
@@ -64,7 +64,7 @@ filterwarnings =
 junit_family=xunit2
 
 [codespell]
-ignore-words-list=ba,blocs,coo,datas,fo,hist,nd,ser
+ignore-words-list=ba,blocs,coo,fo,hist,nd,ser
```

Comment:
is it possible to just ignore pandas/core/strings/accessor.py instead / in-addition to?

Reply:
as in, ignore the whole file? Yes, that's easily done with
```
exclude: ^pandas/core/strings/accessor\.py$
```
in `.pre-commit-config.yaml`, it's just not clear to me why that would be preferrable

Reply:
to avoid the fragility of 'fo'

Reply:
@jreback an alternative would be to change the example to use `'nox'` instead of `'fox'` - which isn't a real animal, but at least then `no` is a real word and we avoid having to ignore an entire file, does that seem OK?

Reply:
i would rather just ignore that example if we can. it IS pretty fragile. its not likely to change much in the future though, so i suppose if you can't ignore that line then just changing the example is prob ok.

Reply:
OK if we use `dog` then there's no failures and we can use a real animal :tada: 

The original PR changed the example actually, before I asked them to change (sorry @chinggg !) so I've added a commit for them - look good now?

Reply:
Yeah! Thanks @MarcoGorelli , it looks good to me.